### PR TITLE
nit: use curly apostrophes throughout the UI

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

**Comment by MatissJanis** [comment]
Can this be enforced with an eslint rule? I can see us adding them back in eventually if there's nothing prohibiting it.

**Comment by j-f1** [comment]
You successfully nerdsniped me :)

I did a little bit of searching and I am seriously shocked that there doesn’t seem to be a rule for this already (but to be fair the number of exceptions and ignores I had to add for it to be useful suggests it might be quite difficult to generalize).

**Comment by j-f1** [comment]
The failing test is just the flaky one now.

File: packages/eslint-plugin-actual/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "eslint-plugin-actual",
+  "private": true,
+  "main": "./lib/index.js",
+  "exports": "./lib/index.js",
+  "type": "commonjs",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "mocha tests --recursive"
```
Comment (MatissJanis) [inline packages/eslint-plugin-actual/package.json:9]: **🥜 nitpick:** ‏any way we could port these to use `jest`? Would be cool to stick to just one type of test-runner in the entire repo.

**Review by MatissJanis — APPROVED** [review]
Very cool

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**